src/BlockTypes/Checkout.php

```diff
@@ -62,6 +62,13 @@ public function render( $attributes = array(), $content = '' ) {
 		if ( ! $data_registry->exists( 'shippingStates' ) ) {
 			$data_registry->add( 'shippingStates', WC()->countries->get_shipping_country_states() );
 		}
+		if ( function_exists( 'get_current_screen' ) ) {
+			$screen = get_current_screen();
+			if ( $screen->is_block_editor() && ! $data_registry->exists( 'shippingMethodsExist' ) ) {
```

Comment:
I'd add a conditional just for `$screen` here before calling methods - sometimes it's not there, depending on how early this is called.

Reply:
Good call. Done in fbf4c438af4b126f1bfdd6c7141e45db73655b64.